README.md

```diff
@@ -12,6 +12,11 @@ This is a simple javascript booklist application. The project tests the use of o
 
 [Live Demo Link](https://themonster2015.github.io/awesome-books/)
 
```

Comment:
- [ ] the live demo link gives a 404 error, kindly fix this

Reply:
thanks for noticing it. We've fixed it!